### PR TITLE
completed script tag-Troy Edwards

### PR DESCRIPTION
**Pull request by TroyEdwards41**
Screenshot 2024-11-18 at 6 31 16 PM

### CODE REVIEW

File: prep-html-script-tag/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <title>HTML-Script-Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added script file via the src attribute!');
```
Comment (shawnkost) [inline prep-html-script-tag/index.html]: This console log should go inside of your `main.js` file, not in-between the script tags

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Troy!

**Comment by TroyEdwards41** [comment]
Shawn I made the changes but I'm not seeing if it reflected them

File: prep-html-script-tag/index.html [hunk 2]
```diff
@@ -7,8 +7,7 @@
     <script>
       console.log('I have added JS to a script tag in the HTML!');
     </script>
-    <script src="main.js"></script>
-    <script>
+    <script src="main.js">
       console.log('I have added script file via the src attribute!');
```
Comment (shawnkost) [inline prep-html-script-tag/index.html]: It looks like you still have your `console.log` directly in-between your opening and closing `<script>` tag. You should have created a `main.js` file, and you need to move this `console.log` into that `main.js` file.

**Review by shawnkost — CHANGES_REQUESTED** [review]

**Comment by TroyEdwards41** [comment]
Screenshot 2024-11-21 at 7 23 41 AM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**